src/Compilers/Core/MSBuildTask/MSBuildTask.csproj

```diff
@@ -80,6 +79,29 @@
       <LastGenOutput>ErrorString.Designer.cs</LastGenOutput>
     </EmbeddedResource>
   </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Build" Version="$(MicrosoftBuildFixedVersion)" />
```

Comment:
I must admit I don't understand the distinction where some versions are "Fixed" and some aren't. I understood the terminology in your old repoutil where those were ones that didn't have to be these same as everything else but the usage seems different here.